src/voters/services/voterOutreach.service.ts

```diff
@@ -123,6 +125,28 @@ export class VoterOutreachService {
       audienceRequest: audience['audience_request'],
     })
 
+    // If type is SMS, create a TextCampaign
+    if (type === 'sms') {
+      try {
+        await this.textCampaignService.createTextCampaign(
+          campaign.id,
+          `SMS Campaign ${new Date(date).toLocaleDateString()}`,
+          message,
+          audience,
+          messagingScript,
+          new Date(date),
+          imageUrl,
+        )
+
+        this.logger.log(`Created TextCampaign for campaign ${campaign.id}`)
+      } catch (error: any) {
+        this.logger.error(
+          `Failed to create TextCampaign: ${error.message}`,
+          error.stack,
+        )
+      }
+    }
```

Comment:
Not sure this `try-catch` is needed. This error will likely just be redundant since the service we're calling will be throwing it's own exceptions, no?

Also.. we never re-throw here, so any exception thrown out of the `createTextCampaign` will just get swallowed here.

Prolly best to just let the exception bubble up..